utils/build/docker/java/spring-boot-payara.Dockerfile

```diff
@@ -31,5 +31,7 @@ RUN chmod +x /app/app.sh
 
 ENV DD_TRACE_HEADER_TAGS='user-agent:http.request.headers.user-agent'
 ENV APP_EXTRA_ARGS="--port 7777"
+# https://docs.hazelcast.com/hazelcast/lat/phone-homes
```

Comment:
The link is broken :cry:, but in any case nice catch!

Reply:
This link works:
https://docs.hazelcast.com/imdg/4.2/phone-homes

Reply:
Fixed.